JsonPatch/PatchExtensions.cs

```diff
@@ -25,6 +25,20 @@ public static T Apply<T>(this JsonPatch patch, T obj)
 		return Apply<T, T>(patch, obj);
 	}
 
+	/// <summary>
+	/// Apply a patch to an object to transform its data.
+	/// </summary>
+	/// <param name="patch">The patch</param>
+	/// <param name="obj">The object</param>
+	/// <param name="options">Serializer options.</param>		
+	/// <typeparam name="T">The type of the object.</typeparam>
+	/// <returns>New instance of patched object</returns>
+	/// <exception cref="InvalidOperationException">Thrown when the patch cannot be applied.</exception>
+	public static T Apply<T>(this JsonPatch patch, T obj, JsonSerializerOptions? options = null)
```

Comment:
Also, can you add a test that uses this, please?

Reply:
Ah, ok. I used this as external extension method, not inside the code and this might run into issues... sorry, did it just online at github. Was not convenient to me without this extension, but I  can leave it in my own codebase and close this here.

Reply:
I'll see if I can incorporate it somehow.